### ISSUE TITLE
Cannot start timer (Error: Parameter "key" required)

### ISSUE THREAD

**Issue by jthelle** [johannesjo/super-productivity#208]
### Version and platform info
 ~  snap list | grep superproductivity                                                                                                          ✔
superproductivity     2.10.8                      337   stable    johannesjo    -
 ~  unity --version                                                                                                                             ✔
unity 7.5.0
 ~  uname -a                                                                                                                                    ✔
Linux LAPDOG 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

### Current Behavior
Press play button on either a task or top right (pomodoro timer)
Error pops up and asks for app reload (Error: Parameter "key" required)

### Steps to Reproduce (for bugs)
1. Fresh install
2. Link to Jira with api key
3. ???
4. Try to start the timer

### Console Output
main.2f13804….js:1 GLOBAL_ERROR_HANDLER Error: Parameter "key" required
    at e.get (main.2f13804….js:1)
    at e.updateValue (main.2f13804….js:1)
    at e.transform (main.2f13804….js:1)
    at Object.updateRenderer (main.2f13804….js:1)
    at Object.updateRenderer (main.2f13804….js:1)
    at Hb (main.2f13804….js:1)
    at Zb (main.2f13804….js:1)
    at qb (main.2f13804….js:1)
    at Hb (main.2f13804….js:1)
    at Zb (main.2f13804….js:1)

### Error Log (Desktop only)
~/.config/superProductivity directory is missing so no log for you <3

**Comment by johannesjo** [comment]
Thanks for reporting! What's your system language?

**Comment by jthelle** [comment]
Thanks for a quick response! App language is English. My system language is English (United States), formats are Ireland.

**Comment by johannesjo** [comment]
Can you consistently reproduce the error? So far I'm unable to do so and I'm wondering what's happening in "???" :)

**Comment by jthelle** [comment]
Yes 100% reproduction rate. What i did at the ??? was just setting up some general settings. This is my first install of the application.

I found the log file in the ~/snap/ folder, the relevant part is here:
[2019-08-21 10:14:21.364] [error] Frontend Error: Error {} Error
    at e.handleError (file:///snap/superproductivity/340/resources/app.asar/dist/main.38262e3e6a3b9272c703.js:1:3760324)
    at file:///snap/superproductivity/340/resources/app.asar/dist/main.38262e3e6a3b9272c703.js:1:358905
    at t.invoke (file:///snap/superproductivity/340/resources/app.asar/dist/polyfills.5d0bba8cc3b1079c6861.js:1:9279)
    at e.run (file:///snap/superproductivity/340/resources/app.asar/dist/polyfills.5d0bba8cc3b1079c6861.js:1:4482)
    at e.runOutsideAngular (file:///snap/superproductivity/340/resources/app.asar/dist/main.38262e3e6a3b9272c703.js:1:348565)
    at e.tick (file:///snap/superproductivity/340/resources/app.asar/dist/main.38262e3e6a3b9272c703.js:1:358849)
    at file:///snap/superproductivity/340/resources/app.asar/dist/main.38262e3e6a3b9272c703.js:1:356761
    at t.invoke (file:///snap/superproductivity/340/resources/app.asar/dist/polyfills.5d0bba8cc3b1079c6861.js:1:9279)
    at Object.onInvoke (file:///snap/superproductivity/340/resources/app.asar/dist/main.38262e3e6a3b9272c703.js:1:347536)
    at t.invoke (file:///snap/superproductivity/340/resources/app.asar/dist/polyfills.5d0bba8cc3b1079c6861.js:1:9219)
[2019-08-21 10:14:21.394] [error] Frontend Error: Error {} Error
    at e.handleError (file:///snap/superproductivity/340/resources/app.asar/dist/main.38262e3e6a3b9272c703.js:1:3760324)
    at Object.next (file:///snap/superproductivity/340/resources/app.asar/dist/main.38262e3e6a3b9272c703.js:1:354647)
    at t.r [as _next] (file:///snap/superproductivity/340/resources/app.asar/dist/main.38262e3e6a3b9272c703.js:1:323893)
    at t.__tryOrUnsub (file:///snap/superproductivity/340/resources/app.asar/dist/main.38262e3e6a3b9272c703.js:1:469988)
    at t.next (file:///snap/superproductivity/340/resources/app.asar/dist/main.38262e3e6a3b9272c703.js:1:469133)
    at t._next (file:///snap/superproductivity/340/resources/app.asar/dist/main.38262e3e6a3b9272c703.js:1:468201)
    at t.next (file:///snap/superproductivity/340/resources/app.asar/dist/main.38262e3e6a3b9272c703.js:1:467872)
    at t.next (file:///snap/superproductivity/340/resources/app.asar/dist/main.38262e3e6a3b9272c703.js:1:549840)
    at t.emit (file:///snap/superproductivity/340/resources/app.asar/dist/main.38262e3e6a3b9272c703.js:1:323674)
    at file:///snap/superproductivity/340/resources/app.asar/dist/main.38262e3e6a3b9272c703.js:1:347854

**Comment by jthelle** [comment]
Also this same error occurs when the application asks what I would like to do with the time when it was idle, and I try to assign the time to a Jira task.

**Comment by johannesjo** [comment]
Thanks you. I fixed some jira related issues in the 2.10.9 release, maybe you can try again with that version?

I've to admit that I'm really in the dark about what causes the error. Could you maybe provide your jira settings (without the credentials of course) and the pomodoro settings, you're using? What also would be interesting, if this happens without the pomodoro timer being enabled or without the jira feature being enabled on the same project as well as on another project.

**Comment by jthelle** [comment]
Disabling Jira integration made the the app not crash (but the Pomodoro timer didnt count). So this lead me to search for a solution in the Jira settings.

I exported all data to a file from the settings and found the key "key" used in the task data structures. It was set to either "indeterminate" or a username ("jukka.helle" in my case).

So, naturally I tried disabling "Check if the currently worked on issue is assigned to current user" in Jira settings, and behold, the timer started functioning!
